sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/CallToAsyncMethodShouldNotBeBlocking.cs

```diff
@@ -98,5 +98,11 @@ public Task<string> Run(Task<string> task)
                 return completedTask.Result; // Noncompliant FP, task is already completed at this point.
             });
         }
+
```

Comment:
Please add this line into `SleepAsync()` to be sure to handle it correctly for the other circumstances
`var name = nameof(Thread.Sleep);`